### PR TITLE
Add auto-complete to the send & receive screen

### PR DESCRIPTION
**Pull request by Tyratox**
Screenshot 2023-01-12 at 16 13 37
Screenshot 2023-01-12 at 16 13 45
Screenshot 2023-01-12 at 16 13 47

### CODE REVIEW

**Comment by Tyratox** [comment]
I just noticed that it is possible to send money to public keys of other roll calls.. Should the suggestions show the attendees of all roll calls?

File: fe1-web/src/core/components/ScannerInput.tsx [hunk 1]
```diff
@@ -4,32 +4,61 @@ import { StyleSheet, View } from 'react-native';
...
+        />
+      </View>
+      {/* <Input
         value={value}
         placeholder={placeholder}
         onChange={onChange}
         enabled={enabled}
         testID={testID}
-      />
+        /> */}
```
Comment (pierluca) [inline fe1-web/src/core/components/ScannerInput.tsx:61]: was this forgotten here?

**Review by pierluca — COMMENTED** [review]
@lhmerino do you see a case where it makes sense to send money to a PoP token that's from a previous roll call ?

Otherwise I'd limit sending to only valid / current PoP tokens.

File: fe1-web/src/core/components/__tests__/ScannerInput.test.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react-native';
+import React from 'react';
+
+import AutocompleteInput from '../AutocompleteInput';
+
+describe('AutocompleteInput', () => {
+  it('renders correctly', () => {
+    const { toJSON } = render(
+      <AutocompleteInput value="" onChange={() => {}} suggestions={['aaa', 'aab', 'aac']} />,
```
[the remark this answers is not in the document]
Comment (Tyratox) [reply fe1-web/src/core/components/__tests__/ScannerInput.test.tsx:9]: uhhh this is a very good question, I thought I did this in a new file 😨😂

File: fe1-web/src/core/components/ScannerInput.tsx (hunk 1 above)
Comment (Tyratox) [reply fe1-web/src/core/components/ScannerInput.tsx:61]: Yep, that it was 😅

**Review by pierluca — APPROVED** [review]

**Pull request merged**